runtime/src/crypto.rs

```diff
@@ -0,0 +1,128 @@
+// Licensed under the Apache-2.0 license
+
+use caliptra_common::{crypto::Ecc384KeyPair, keyids::KEY_ID_TMP};
+use caliptra_drivers::{
+    hmac384_kdf, Array4x12, Ecc384PrivKeyOut, Ecc384PubKey, Hmac384Data, Hmac384Key, Hmac384Tag,
+    KeyId, KeyReadArgs, KeyUsage, KeyWriteArgs,
+};
+use caliptra_error::CaliptraResult;
+use zerocopy::AsBytes;
+use zeroize::Zeroize;
+
+use crate::Drivers;
+
+pub enum Crypto {}
```

Comment:
why enum instead of struct?

Reply:
It was the pattern I see on other crypto implementations in [FMC](https://github.com/chipsalliance/caliptra-sw/blob/e61eb66bfcd42e56438c1548272c7b568ce9f8d0/fmc/src/flow/crypto.rs#L16) and [ROM](https://github.com/chipsalliance/caliptra-sw/blob/e61eb66bfcd42e56438c1548272c7b568ce9f8d0/rom/dev/src/flow/cold_reset/crypto.rs#L50). Wasn't sure what was more idiomatic.

Reply:
After reading up it looks like this pattern prevents someone from making an instance of a `Crypto`.